src/router/index.js

```diff
@@ -10,12 +10,12 @@ const router = createRouter({
       component: HomeView
     },
     {
-      path: '/about',
-      name: 'about',
+      path: '/todo',
```

Comment:
Good use of routes. Feel free to remove the route for the HomeView component and set the default route to your TodoListView.
```
path: '/',
name: 'todolist',
component: () => import('../views/TodoListView.vue')
```